src/components/cardSong/index.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useState, useEffect } from "react";
+import { dataUpload } from "../../services/requetsToEndpoint";
+import Track from "../../models/track.inteface";
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+
+// Creamos una prop para pasar el término de búsqueda
+interface CardSongProps {
```

Comment:
create a file src/components/cardSong/cardSong.interface.ts and set the interface there